.github/workflows/deploy_hooks.yml

```diff
@@ -0,0 +1,72 @@
+name: Deployment Webhooks
+
+on:
+  deployment_status
```

Comment:
It might be worth an update to add `workflow_dispatch` and possibly a default or define a few custom inputs (with defaults) to make testing it easier going forward 

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#providing-inputs

Reply:
I added the workflow_dispatch here: https://github.com/djangocon/2023.djangocon.us/pull/123

Since can add labels to a PR afterwards, there's no real need to allow the SHA to be customizable. We can trigger it in either configuration.